packages/entity-detail/src/components/SubGrid/SubGrid.js

```diff
@@ -2,13 +2,15 @@ import PropTypes from 'prop-types'
 import React from 'react'
 import EntityListApp from 'tocco-entity-list/src/main'
 import styled from 'styled-components'
+import {viewPersistor} from 'tocco-util'
 
 const StyledListApp = styled.div`
   width: 100%;
 `
 
 const SubGrid = props => {
   const formBase = `${props.detailFormName}_${props.formField.path}`
+  const subgridPersistorId = `subgrid-${props.entityName}-${props.entityKey}-${formBase}-`
```

Comment:
- is this unique enough? should `props.appId` also be part of that store id? (see line 17)
(- `-` at the end of the id not necessary)